test/acceptance/framework/config/config.go

```diff
@@ -87,7 +88,16 @@ func (t *TestConfig) entImage() (string, error) {
 		return "", err
 	}
 
-	return fmt.Sprintf("hashicorp/consul-enterprise:%s-ent", chartMap["appVersion"]), nil
+	appVersion := chartMap["appVersion"].(string)
```

Comment:
This cast is probably safe here since we know it's always a string, but I'd still check and error just in case.